### PR TITLE
feat: add ciskip option

### PR DESCRIPTION
**Pull request by wdconinc**
### Briefly, what does this PR introduce?
Option to skip CI runs on push.

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [x] New feature (issue #__)
- [ ] Documentation update
- [ ] Other: __

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@ case "${GITHUB_EVENT_NAME}" in
 push|create|pull_request|workflow_dispatch)
   git-setup
   git fetch --all
-  git push -f --all target
+  git push -f ${INPUT_SKIPCI:+-o skip.ci} --all target
```
Comment (veprbl) [inline entrypoint.sh]:
```suggestion
  git push -f ${INPUT_SKIPCI:+-o ci.skip} --all target
```
I believe, `skip.ci` was always supported (at least back in the day on Travis), but only for typo-friendliness.
https://docs.gitlab.com/ee/user/project/push_options.html

File: entrypoint.sh [hunk 2]
```diff
@@ -17,7 +17,7 @@ case "${GITHUB_EVENT_NAME}" in
 push|create|pull_request|workflow_dispatch)
   git-setup
   git fetch --all
-  git push -f --all target
+  git push -f ${INPUT_SKIPCI:+-o ci.skip} --all target
```
Comment (wdconinc) [inline entrypoint.sh]:
```suggestion
  git push -f ${INPUT_CISKIP:+-o ci.skip} --all target
```

File: action.yml [hunk 3]
```diff
@@ -25,4 +28,5 @@ runs:
         INPUT_TARGET_URL: ${{ inputs.url }}
         INPUT_TARGET_TOKEN: ${{ inputs.token }}
         INPUT_TARGET_USERNAME: ${{ inputs.username }}
+        INPUT_SKIPCI: ${{ inputs.skipci }}
```
Comment (wdconinc) [inline action.yml]:
```suggestion
        INPUT_CISKIP: ${{ inputs.ciskip }}
```

File: action.yml [hunk 4]
```diff
@@ -16,6 +16,9 @@ inputs:
   token:
     description: "GitLab token"
     required: true
+  skipci:
```
Comment (wdconinc) [inline action.yml]:
```suggestion
  ciskip:
```

**Review by veprbl — APPROVED** [review]

**Comment by wdconinc** [comment]
Hard to put a CI test for this in...

**Comment by wdconinc** [comment]
Tested in https://eicweb.phy.anl.gov/EIC/benchmarks/detector_benchmarks/-/merge_requests/160.

**Pull request merged**